### ISSUE TITLE
Gradle 7 support

### ISSUE THREAD

**Issue by jpschewe** [TheBoegl/gradle-launch4j#130]
Per issue #88 it has been noted that gradle 7 isn't supported yet as the tests won't run.

**Comment by jpschewe** [comment]
@TheBoegl  check out https://github.com/jpschewe/gradle-launch4j/tree/gradle-7 for a start on the gradle 7 support. I have tests running, although failing. However that might be due to a different environment.

**Comment by TheBoegl** [comment]
Thanks for your effort. I am very busy at the moment but will look at your branch after Christmas.

**Comment by TheBoegl** [comment]
@jpschewe I've merged your changes and fixed a few bugs in your code but unfortunately if forgot that the groovy update to version 3 breaks compatibility with all gradle versions below 7. Further details can be found in the gradle issue linked in #118 . To simplify further testing I've pushed the necessary changes to the branch `gradle-7`. The failing tests can be found [here](https://ci.appveyor.com/project/TheBoegl/gradle-launch4j/builds/45784170).
As we are still targeting gradle 6 (and earlier) with this major version and I don't see a backwards compatible way to upgrade to gradle 7 at the moment, I'm closing this issue.
